docs/Dockerfile

```diff
@@ -13,18 +13,15 @@ RUN groupadd jenkins --gid=$GID -o && useradd jenkins --uid=$UID --gid=$GID --cr
 RUN (mkdir -p /var/lib/teleport && chown -R jenkins /var/lib/teleport)
 
 # Install SASS
-RUN (apt-key update && \
-             apt-get -q -y update --fix-missing && \
-             apt-get -q -y update && \
-             apt-get install -q -y ruby-sass)
-             
-ENV LANGUAGE="en_US.UTF-8" \
-    LANG="en_US.UTF-8" \
-    LC_ALL="en_US.UTF-8" \
-    LC_CTYPE="en_US.UTF-8" \
+RUN apt-get update -y && apt-get install -q -y ruby-sass parallel
+
+# Install Go & milv, go version doesn't particularly matter as it isn't used in the product
+RUN mkdir -p /opt /gopath
+ENV GOROOT="/opt/go" \
     GOPATH="/gopath" \
-    GOROOT="/opt/go" \
     PATH="$PATH:/opt/go/bin:/gopath/bin"
+RUN cd /opt && curl https://dl.google.com/go/go1.14.linux-amd64.tar.gz | tar xz
+RUN GO111MODULE=on go get -u -v github.com/magicmatatjahu/milv@v0.0.6
```

Comment:
How about move all things only relevant for building the prerequisites (like Go for building milv here) to the build container (perhaps as a multistage build) and have the docs container without it?

Reply:
Good idea.  I'll look into this.

Reply:
Great suggestion.  The Dockerfile is considerably simpler after I implemented this change in 
9c8fd4a.